adapter/src/main/java/com/twilio/kudu/sql/rules/KuduSortRule.java

```diff
@@ -75,6 +81,7 @@ public boolean canApply(final RelTraitSet sortTraits, final KuduQuery query, fin
     }
 
     int pkColumnIndex = 0;
+    pkSortColumns.clear();
```

Comment:
since we just initialized `pkSortColumns` on line 63,  is this needed ?

Reply:
This rule object is initialized once and called multiple times so we need the clear.